components/hab/src/command/studio/enter.rs

```diff
@@ -13,12 +13,39 @@ use crate::{config,
 
 pub const ARTIFACT_PATH_ENVVAR: &str = "ARTIFACT_PATH";
 
+const AUTH_TOKEN_ENVVAR: &str = "HAB_AUTH_TOKEN";
+const BLDR_URL_ENVVAR: &str = "HAB_BLDR_URL";
+const CTL_SECRET_ENVVAR: &str= "HAB_CTL_SECRET";
 const ORIGIN_ENVVAR: &str = "HAB_ORIGIN";
 const STUDIO_CMD: &str = "hab-studio";
 const STUDIO_CMD_ENVVAR: &str = "HAB_STUDIO_BINARY";
 const STUDIO_PACKAGE_IDENT: &str = "core/hab-studio";
 
 pub fn start(ui: &mut UI, args: &[OsString]) -> Result<()> {
+    if henv::var(AUTH_TOKEN_ENVVAR).is_err() {
+        let config = config::load()?;
+        if let Some(auth_token) = config.auth_token {
+            debug!("Setting {}={} via config file", AUTH_TOKEN_ENVVAR, &auth_token);
```

Comment:
Seems like, with the exception of the origin--and maybe we make the origin match--the debug output is "Setting <env var name>=<env var value>", this could be abstracted to a `env_debug_output()`?

Reply:
we did.

Reply:
Making them all match (with the exception of not printing the value of `HAB_AUTH_TOKEN`) sounds good.